imagersite/imager_images/forms.py

```diff
@@ -0,0 +1,43 @@
+from .models import *
+from django.forms import ModelForm
+from django.contrib.gis import forms as geoforms
+# import floppyforms as geoforms
+
+class AlbumForm(ModelForm):
+    class Meta:
+        model = Album
+        fields = ['title', 'description', 'privacy', 'cover', 'photos']
+
+
+class PhotoForm(ModelForm):
+    class Meta:
+        model = Photo
+        fields = ['title', 'description', 'privacy', 'file']
+
+
+class LocationForm(geoforms.Form):
+    class Meta:
+        model = Photo
+        fields = ['geom']
+    # point = geoforms.PointField()
+    point = geoforms.PointField(widget=
+        geoforms.OSMWidget(attrs={'map_width': 800, 'map_height': 500}))
```

Comment:
Ok, good to note! It was actually lifted verbatim out of the django docs [here](https://docs.djangoproject.com/en/1.8/ref/contrib/gis/forms-api/#django.contrib.gis.widgets.BaseGeometryWidget.template_name).